frame/support/src/lib.rs

```diff
@@ -65,6 +65,13 @@ pub use sp_std;
 #[doc(hidden)]
 pub use tt_call::*;
 
+// Re-exports go here:
+#[doc(hidden)]
+pub mod private {
```

Comment:
```suggestion
pub mod __private {
```

Maybe to make it even more "weird" to use this :P